### PR TITLE
some preparatory work before introducing a package.

### PR DESCRIPTION
**Pull request by stevecooperorg**
Three commits here, each doing a little bit, and each one commented.

I'm in the process of trying to extract out a style --- so you can do `\usepackage[modern]{rpg}` or `\usepackage[fantasy]{rpg}` --- but in doing so I needed to get a few things off.

Really this is so that I've actually raised a first PR into the repo...

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,18 +1,23 @@
...
 all: doc
 doc: pdf
+pdf: PREVIEW=
 pdf: $(DOCNAME).pdf
 
 # Rules
-%.pdf: %.tex
-	$(LATEXMK) -auxdir=./build -pdfxe -M -MP -MF $*.d $*
+%.pdf: %.tex FORCE
+	$(LATEXMK) -xelatex -interaction=nonstopmode $(PREVIEW) $*
```
Comment (kierun) [inline Makefile:20]: `-auxdir=./build` makes sure all the build files are not polluting the top level. Could we keep that?

File: Makefile [hunk 2]
```diff
@@ -23,11 +28,20 @@ clean: mostlyclean
...
 
-format:
-	latexindent -w -s *.tex
-	${RM} *.bak[0-9]
+format: indent/
+	# format all tex and sty files; stick the cruft in the indent/ directory, then delete the cruft
+	latexindent $(wildcard *.tex) $(wildcard src/*.tex) $(wildcard *.sty) \
+		--local --overwriteIfDifferent --silent --modifylinebreaks --cruft=indent/
+	# remove all backup files created by latexindent
+	$(FIND) indent -name "*.bak*" -maxdepth 1 -delete
```
Comment (kierun) [inline Makefile:36]: Could you replace `"*.bak*" ` with "*.bak[0-9]" as it make it less greed?

**Comment by kierun** [comment]
@stevecooperorg Thank you for the PR. It is much appreciated. :heart: 

I have been using [git conventional commit](https://www.conventionalcommits.org/en/v1.0.0/), which could be overkill for this. ¯\\\_(ツ)\_/¯

**Review by kierun — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (stevecooperorg) [reply Makefile:20]: I added this to .latexmkrc as 

```
$aux_dir = 'build';
```

I'm finding that the rc file's quite neat, especially as you add more documents and targets, to give consistent builds.

**Pull request closed without merging**